src/lib/strings/url-helpers.ts

```diff
@@ -162,7 +162,7 @@ export function convertBskyAppUrlIfNeeded(url: string): string {
   if (isBskyAppUrl(url)) {
     try {
       const urlp = new URL(url)
-      return urlp.pathname
+      return urlp.pathname + urlp.search
```

Comment:
I'm not sure this is a safe change — some code calling `convertBskyAppUrlIfNeeded` assumes there's no search. What do you think of the callsites? Could make a separate function for the case of handling in-app links. Or could return a richer data structure and let the caller decide which parts to use.